etn-ccis/blui-react-workflows
# Feature/4145 registration workflow component

Fixes # BLUI-4145



#### Changes proposed in this Pull Request:

- Added  registration workflow component



#### Screenshots / Screen Recording (if applicable)
https://github.com/etn-ccis/blui-react-workflows/assets/120575281/025eb68a-5661-4b75-a99e-42d72976dc48



#### To Test:

- yarn start:example 2
- Click on Registration Workflow
- Enter text in the field and click Next
- Enter text in the field and click Previous
- Play around with screens to see the saved data



#### Any specific feedback you are looking for?

- I will add the example for this component and will open this PR, need to pull in more priority ticket, hence keeping this one draft.
